concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -2304,6 +2305,132 @@ jobs:
         params:
           release: pipeline-pool
 
+  - name: reset-cloudfoundry-passwords
```

Comment:
If we run this job twice and forget the manual step of updating Grafana password in the UI, then we lose the old Grafana password and can't log in. I don't know what recovery of that password is possible, would have to look under the hood of Grafana.